src/main/java/seedu/address/ui/CustomerWindow.java

```diff
@@ -0,0 +1,125 @@
+package seedu.address.ui;
+
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import java.util.logging.Logger;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.web.WebView;
+import javafx.stage.Stage;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.person.Customer;
+
+/**
+ * Controller for a help page
+ */
+public class CustomerWindow extends UiPart<Stage> {
+
+    public static final String DEFAULT_MESSAGE = "Customer Window";
+
+    private static final String CODED = "QUl6YVN5QmtSOTlwR0EwSElTUFQxSFA2d1NLOFd2cTBPZGdCbU9J";
+    private static final String WEBVIEW_WRAPPER = "<html style=\"background: #424242;\">%s</html>";
+    private static final String MAPS_WRAPPER = "<iframe width=\"100%%\" height=\"330\" frameborder=\"0\" "
+            + "style=\"border:0\" src=\"https://www.google.com/maps/embed/v1/search?q=%s&key=%s\" "
```

Comment:
maybe can write a javadocs for these html stuffs

Reply:
i abstracted all these tgt with the function to produce the address into another class

Reply:
done